documentation/operations/archival-process.md

```diff
@@ -0,0 +1,23 @@
+# Archival Process
+
+To archive the instances you will need to connect to the Ocim server using ssh and use the Django shell to archive instances since there is currently no UI to do so. Please, don't wait for the archival process for more than 5 minutes once it's running; this takes some hours to run, so it's better to check back later.
+
+*  Connect to Ocim using: `ssh ubuntu@ocim.net.opencraft.hosting`.
+*  See a list of screens with `screen -list`.
+*  The one that ends with "console" is what we want: `screen -r 1240..ocim` (or similar Ocim screen name).
+*  There are multiple screen windows in that session including one that runs the Ocim app in the foreground. Be careful not to fiddle with it or kill it by accident.
+*  You can switch around screens with CTRL+a -> SHIFT+" (press control and 'a' together, then shift and the double-quote key).
+*  Select one until you find one where you're logged in as `www-data` in the `~/opencraft` folder with the `opencraft` venv activated.
+*  Run the following command to archive the instances: `make manage 'archive_instances --domains=sandbox.example.com,test.example.com'`.
+*  You can leave the screen session with CTRL+a -> d (control and 'a' together, then 'd').
+
+If the `screen` session key combinations are too complex or there's some other issue, for now you can do the following, but eventually you should get familiar with how `screen` works since you'll need it for redeployments:
+
+*  Connect to Ocim using: `ssh ubuntu@ocim.net.opencraft.hosting`.
+*  Switch to the www-data user using: `sudo -Hu www-data bash`.
+*  Go to the Ocim directory at `/var/www/opencraft`.
+*  Run the following command to archive the instances: `make manage 'archive_instances --domains=sandbox.example.com,test.example.com'`.
```

Comment:
We should use a screen either way because the downside with the approach is, if somehow you get disconnected the process will get killed, we wouldn't want that to happen right!

Reply:
I copied this from the ticket, since this task is newcomer friendly I guess this was added but can be removed

Reply:
This was removed on latest version